src/script/hydra-notify

```diff
@@ -93,11 +93,13 @@ my $task_dispatcher = Hydra::TaskDispatcher->new(
 my $dbh = $db->storage->dbh;
 
 my $listener = Hydra::PostgresListener->new($dbh);
+$listener->subscribe("build_finished");
```

Comment:
Super-nit: IMHO, the order should be "queued" -> "started" -> "finished", since that's the logical path that builds will take (to the best of my understanding, that is).

Reply:
This was sort of true before this patch, but it gets a bit complicated about ordering, so I just sorted them alphabetically. Does that sound ok?

Also, I'm closing this PR and re-opening it against upstream.

Reply:
Fair enough.